core/src/test/java/org/everit/json/schema/loader/ObjectSchemaLoaderTest.java

```diff
@@ -56,12 +56,16 @@ public void objectWithAdditionalPropSchema() {
     public void objectWithPropDep() {
         ObjectSchema actual = (ObjectSchema) SchemaLoader.load(get("objectWithPropDep"));
         assertEquals(1, actual.getPropertyDependencies().get("isIndividual").size());
+        actual = (ObjectSchema) loadAsV201909(get("objectWithPropDep_V201909"));
```

Comment:
This can be movd into a separate tests, so that each test contains only one given-when-then triad (same for the other test changes below).

Reply:
Ok, at first I used separate tests then I changed my mind :)